addons/core/translations/resources/en-us.yaml

```diff
@@ -256,3 +256,4 @@ target:
       add:
         title: Add Host Sets
         description: Select host sets to assign to this target.
+    secret_label: ■ ■ ■ ■ ■ ■ ■ ■ ■ ■ ■ ■
```

Comment:
Not sure if this is the bet translation file to keep this string. Thoughts?

Reply:
What do you think about putting it in https://github.com/hashicorp/boundary-ui/blob/main/addons/core/translations/form/en-us.yaml?